src/sas/qtgui/Utilities/UnitTesting/GuiUtilsTest.py

```diff
@@ -46,12 +46,6 @@ def testGetAppDir(self):
         """
         pass
 
-    def testGetUserDirectory(self):
```

Comment:
Suggestion: Move this test into your config tests.

Reply:
Maybe if the tests are refactored more generally so that everything in system is in the same place.

Reply:
oh, actually, this is part of the gui testing suite, which is currently a very different place to the config tests.